packages/rollup-plugin-minify-html-literals/package.json

```diff
@@ -3,10 +3,116 @@
   "version": "0.0.0",
   "description": "Rollup plugin to minify HTML template literal strings",
   "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "types": "index.d.ts",
   "author": "Google LLC",
   "license": "BSD-3-Clause",
-  "private": true
+  "private": true,
+  "type": "module",
+  "files": [
+    "index.*",
+    "/lib/"
+  ],
+  "keywords": [
+    "rollup-plugin",
+    "minify",
+    "html",
+    "literal",
+    "literals",
+    "template",
+    "tagged",
+    "lit-html"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lit/lit.git",
+    "directory": "packages/rollup-plugin-minify-html-literals"
+  },
+  "scripts": {
+    "clean": "wireit",
+    "build": "wireit",
+    "test": "wireit",
+    "lint": "prettier --write \"*.{js,ts,json,md}\" \"src/**/*\"",
+    "precommit": "lint-staged"
+  },
+  "wireit": {
+    "build": {
+      "command": "tsc && mkdir lib/types/ && cp -r src/lib/types/ lib/types/",
+      "clean": "if-file-deleted",
+      "dependencies": [
+        "clean"
+      ],
+      "files": [
+        "src/**/*.ts",
+        "tsconfig.json"
+      ],
+      "output": [
+        "index.{js,js.map,d.ts,d.ts.map}",
+        "lib/**/*.{js,js.map,d.ts,d.ts.map}",
+        "test/**/*.{js,js.map,d.ts,d.ts.map}",
+        "tsconfig.tsbuildinfo"
+      ]
+    },
+    "clean": {
+      "command": "rimraf {lib/,test/,} && rimraf tsconfig.tsbuildinfo && rimraf index.{js*,d.ts,d.ts.map,} && rimraf {.nyc_output/,}"
+    },
```

Comment:
with outputs set on wireit configs, we can drop this script and remove `rimraf` dependency too.